core/curl-websocket-utils.c

```diff
@@ -157,6 +157,13 @@ _cws_header_has_prefix(const char *buffer, const size_t buflen, const char *pref
     return strncasecmp(buffer, prefix, prefixlen) == 0;
 }
 
+#if 0
+/* The __BYTE_ORDER__ define is GCC-specific.
```

Comment:
Even if you don't define the `__BYTE_ORDER__` macro, it'll fall-through to little-endian, which is not exactly good. The endian-agnostic issue here is superior.